flatpak/nl.jknaapen.fladder.yaml

```diff
@@ -0,0 +1,216 @@
+id: nl.jknaapen.fladder
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: Fladder
+finish-args:
+  # X11/Wayland access
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  # Audio playback
+  - --socket=pulseaudio
```

Comment:
Would this mean it only works on pulseaudio, not with pipewire?

Reply:
That's a fair point, apparently it works if there's the pipewire-pulseaudio compat layer, but we may need to do something like done in this PR https://github.com/flathub/com.github.iwalton3.jellyfin-mpv-shim/pull/10 .
I'll investigate it later

Reply:
I added the changes to allow mpv's access to the pipewire socket. I can confirm with `pw-cli ls Node` that the previous flatpak would use `client.api=pipewire-pulse`, whereas with this change it uses pipewire directly. Both cases worked as expected.

I've also added a title to the media_kit `PlayerConfiguration` so that the app shows as `Fladder` in the volume mixer and not the default `mpv`. The icon is still mpv's in the volume mixer, I could not find what causes it. But I guess it may not be worth digging into, especially with #122

Reply:
Thanks for implementing checkin this out. Always a good idea to support mutliple use cases.

> I've also added a title to the media_kit `PlayerConfiguration` so that the app shows as `Fladder` in the volume mixer and not the default `mpv`. The icon is still mpv's in the volume mixer, I could not find what causes it. But I guess it may not be worth digging into, especially with #122

Correct best to wait after that issue is closed whether this is something we want/need fixed.